Task_3.txt

```diff
@@ -0,0 +1,8 @@
+Write bash command(s) to read content of "test.txt" file, change all "test" keywords with "DONE" keyword and write in "test.result.txt" file.
+
+
+1. Open terminal
+2. Go to the corresponding place -> cd Desktop/pdp_training command
+3. Enter while read line; do echo $line; done < test.txt command
+4. Enter sed 's/testing/DONE/g; /test/DONE/g; s/Test/DONE/g' test.txt > test.result.txt command
+
```

Comment:
Astghik jan 
 - executed command from line 3 and it shows content for test.txt file
 - executed command from line 4 and it fails with error 'sed: 1: "s/testing/DONE/g; /test ...": extra characters at the end of D command'

Could you please remove all extra words from solution and keep just solution in 'Task_3.txt' file.